docs/notes-KZG10.md

```diff
@@ -1,3 +1,74 @@
 In this module we show how and why the KZG10 
-polynomial commitment scheme has been modularised 
-for this PLONK implementation.
\ No newline at end of file
+polynomial commitment scheme has been implemented
+for this PLONK implementation.
+
+KZG10 Commitments 
+==================
+
+PLONK can be constructed with different 
+commitment schemes and does not requre solely
+homomorphic commitments. However, this library
+has only homomorphic commitment schemes as they 
+are intelligble for users and have many useful
+properties. 
+
+In addition to the code complexity issue, 
+described immediately above, there is  
+another reason for choosing a commitment
+scheme with homomorphism. PLONK makes use
+of the linearisation technique, originally
+concieved in the SONIC [paper][sonic_paper]. 
+This technique requires the the commitment 
+scheme to be homomorphic. Without modification 
+of PLONK by means of remving the lineariser, 
```

Comment:
nit: removing

Reply:
Nit: 

Can you rewrite this part to flow a bit better? In particular "Without modification of PLONK by means of remving the lineariser, we are unable to use techniques such as 
 [FRI][fri_paper] protocol. "